src/index.html

```diff
@@ -5,8 +5,353 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="container">
+      <a href="#home" class="photo">
+        <img 
+          src="./images/logo.png" 
+          alt="icon Moyo" 
+          class="photo__logo-Moyo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#" class="is-active">APPLE</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">SAMSUNG</a></li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#" class="is-active">APPLE</a>
          </li>

          <li class="nav__item">
            <a href="#" class="nav__link">SAMSUNG</a>
          </li>
```
Now it looks better and cleaner 

Fix everywhere